collectors/nodestats/pipeline_subcollector.go

```diff
@@ -84,6 +92,13 @@ func (collector *PipelineSubcollector) Collect(pipeStats *responses.SinglePipeli
 		ch <- prometheus.NewMetricWithTimestamp(*pipeStats.Reloads.LastFailureTimestamp, prometheus.MustNewConstMetric(collector.ReloadsLastFailureTimestamp, prometheus.GaugeValue, 1, pipelineID))
 	}
 
+	if pipeStats.Reloads.LastSuccessTimestamp != nil {
```

Comment:
Also, it'd be better to have this metric always exist, rather than only exist when the timestamps are non-null in Logstash. The main issue is what timestamp should be used, as there is no `nil` time in Go. Should "Last success" be `Time.Now()`. The gauge value should be `0` I suppose.

Reply:
Oh I missed it. Could you fix that? I am currently walking to play squash so I will be AFK for an extended period of time